addons/l10n_ar_withholding/models/afip.py

```diff
@@ -0,0 +1,34 @@
+from odoo import models, fields
+
+
+class AfipEarningsTableScale(models.Model):
+    _name = 'afip.earnings_table.scale'
+    _description = 'afip.earnings_table.scale'
+
+    name = fields.Char(required=True)
+    line_ids = fields.One2many('afip.earnings_table.scale.detail', 'scale_id')
+
+
+class AfipEarningsTableScaleDetail(models.Model):
+    _name = 'afip.earnings_table.scale.detail'
+    _description = 'afip.earnings_table.scale.detail'
+    _order = 'importe_desde, importe_hasta'
+
+    scale_id = fields.Many2one(
+        'afip.earnings_table.scale', required=True, ondelete='cascade',
+    )
+    importe_desde = fields.Float(
```

Comment:
English?  And maybe we can use Monetary fields?

Reply:
@jco-odoo why could we use Monetary fields? these fields are only used to do calculations and I understand that these amounts should not be associated with a currency because the taxable amount must be always in ARS (argentinean currency). If a payment has other currency different than ARS we need to convert the taxable amount to amount in argentinean currency. We can talk more about this point in the Follow up meeting. @vbe-odoo

Reply:
Change to monetay and compute field